backend/medtagger/repositories/users.py

```diff
@@ -1,6 +1,7 @@
 """Module responsible for definition of Users' Repository."""
 from typing import List, Optional
 
+from medtagger.api.utils import get_current_user
```

Comment:
Repositories shouldn't use API namespace. These functions can be used outside of the API scope, eg. in Celery workers that won't have knowledge about logged in user ;)